docs/src/reference/mount.md

```diff
@@ -266,6 +271,36 @@ When running in write mode, the lakeFS URI must point to a branch, not a commit
 
 ---
 
+## Everest for Windows (Cloud Filter API)
+
+Starting at version <version>, Everest mount is available for Windows OS using *Cloud Filter API* (CFAPI)
+Currently, Everest for Windows supports read-only operation.
+
+### Requierments
+
+Everest Mount supports the windows's native Cloud Filter API. No need in additional installations. 
```

Comment:
Please add [link](https://learn.microsoft.com/en-us/windows/win32/cfapi/cloud-files-api-portal) to CFAPI docs. 
Please mentioned minimal version: `Starting in Windows 10, version 1709`